server/etcdserver/server.go

```diff
@@ -1809,6 +1809,11 @@ func (s *EtcdServer) apply(
 			var cc raftpb.ConfChange
 			pbutil.MustUnmarshal(&cc, e.Data)
 			removedSelf, err := s.applyConfChange(cc, confState, shouldApplyV3)
+			if err != nil {
+				s.lg.Error("failed to apply conf change", zap.Bool("shouldApplyV3", bool(shouldApplyV3)), zap.Error(err))
+			} else {
+				s.KV().Commit()
```

Comment:
is it possible to call `Commit` in `applyConfChange`? 

And I think it's better to add comment on `MustSaveMemberToBackend`, `TrimClusterFromBackend` and `MustDeleteMemberFromBackend` and says that please ensure it should be synced with tx buffer. 

It can save time during debugging test cases 😂

Reply:
Yes, makes sense. Suggestion accepted. Updated to get the `ForceCommit` included in `MustDeleteMemberFromBackend` so as to avoid introducing similar issue by other contributors.